modules/workbench/modules/egress_detection/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable aou_env {
+  description = "Short name (all lowercase) of All of Us Workbench deployed environments, e.g. local, test, staging, prod."
+  type        = string
+}
+
+variable sumologic_egress_thresholds {
+  description = "Configuration values for egress search content in SumoLogic. The name (key) describes simply the tier name and config: "
+  type        = map(map(any))
+}
+
+variable sumologic_parent_folder_id_hexadecimal {
```

Comment:
Should document somewhere that the parent folder and webhook are to be generated manually outside of Terraform